service/src/main/java/uk/gov/hmcts/reform/fpl/utils/PeopleInCaseHelper.java

```diff
@@ -21,4 +23,12 @@ public static String getFirstRespondentLastName(List<Element<Respondent>> respon
             .map(RespondentParty::getLastName)
             .orElse("");
     }
+
+    //To be deleted when print and post is available to use (before March 3rd)
```

Comment:
do we have a jira to track this or do we need one?

Reply:
https://tools.hmcts.net/jira/browse/FPLA-1287 comment isn't really necessary so I will delete it